plugin/src/main/java/org/opensearch/ml/action/register/TransportRegisterModelAction.java

```diff
@@ -233,6 +233,7 @@ private void registerModel(MLRegisterModelInput registerModelInput, ActionListen
                 throw new IllegalArgumentException("URL can't match trusted url regex");
             }
         }
+        System.out.println("registering the model");
```

Comment:
Are we keeping this?

Reply:
Let's avoid using system out print

Reply:
Yes I guarante these redundant lines will be removed in my next PR.